Create explorations

devorahrena opened the pull request:
[Handout-Pigeonhole1.pdf](https://github.com/PrimedMinds/primedminds.github.io/files/888560/Handout-Pigeonhole1.pdf)
[Handout-Pigeonhole2.pdf](https://github.com/PrimedMinds/primedminds.github.io/files/888559/Handout-Pigeonhole2.pdf)

devorahrena committed:
Create explorations

drs0611 commented:
[Task-Pigeonhole1.pdf](https://github.com/PrimedMinds/primedminds.github.io/files/888564/Task-Pigeonhole1.pdf)
[Task-Pigeonhole2.pdf](https://github.com/PrimedMinds/primedminds.github.io/files/888565/Task-Pigeonhole2.pdf)
[Task-Pigeonhole3.pdf](https://github.com/PrimedMinds/primedminds.github.io/files/888566/Task-Pigeonhole3.pdf)

devorahrena committed:
Add files via upload

devorahrena committed:
Delete explorations

devorahrena committed:
Create explorations

devorahrena committed:
upload Handout-Pigeonhole1.pdf

devorahrena committed:
Delete Handout-Pigeonhole1.pdf

devorahrena committed:
Add files via upload

Add Handout-Pigeonhole2, Task-Pigeonhole-1, Task-Pigeonhole-2, Task-Pigeonhole-3

devorahrena committed:
Delete explorations

devorahrena committed:
Delete Handout-Pigeonhole2.pdf

devorahrena committed:
Delete Task-Pigeonhole1.pdf

devorahrena committed:
Delete Task-Pigeonhole2.pdf

devorahrena committed:
Delete Task-Pigeonhole3.pdf

devorahrena committed:
Create map

add map code, works for all maps

devorahrena committed:
Rename map to map.js

devorahrena committed:
Rename public/javascript/map.js to javascript/map.js

devorahrena committed:
Create pigeonhole.html

devorahrena committed:
Add files via upload

devorahrena committed:
Add files via upload

devorahrena committed:
Update pigeonhole.html

devorahrena committed:
Update pigeonhole.html

devorahrena committed:
Update pigeonhole.html

devorahrena committed:
Update pigeonhole.html

devorahrena committed:
Update pigeonhole.html

devorahrena committed:
Update pigeonhole.html

devorahrena committed:
Rename maps/pigeonhole.html to pigeonhole.html

devorahrena committed:
Update pigeonhole.html

devorahrena committed:
Update pigeonhole.html

devorahrena committed:
Update pigeonhole.html

devorahrena committed:
Add files via upload

devorahrena committed:
Update and rename pigeonhole.html to konigsberg.html

devorahrena committed:
Rename pigeonhole.css to konigsberg.css

devorahrena committed:
Update pp_flow.js
